python/lsst/ts/audiotrigger/laser_alignment_listener.py

```diff
@@ -165,30 +154,34 @@ def __init__(
             )
         )
         self.start_laser_task = utils.make_done_future()
+        self.log_server = tcpip.OneClientServer(
+            host=tcpip.LOCAL_HOST, port=18840, log=self.log
```

Comment:
Please consider to put the port number into the configuration file instead of hard-coding it here.

Reply:
I could put it as a command line argument. I don't have a config file for this since it doesn't use any kafka components.